### ISSUE TITLE
excludeGlob not working

### ISSUE THREAD

**Issue by mtskf** [elwin013/parcel-plugin-static-files-copy#36]
```
"staticFiles": {
  "staticPath": [
    {
      "staticPath": "src/img",
      "staticOutDir": "img",
      "watcherGlob": ["**/*.jpg", "**/*.png", "**/*.svg"]
    },
    {
      "staticPath": "src",
      "excludeGlob": "**/!(locales)/*.+(!(txt)|!(json))"
    }
  ]
}
```

All the files are copied...

**Comment by giannif** [comment]
@mtskf #35 fixes this, I've been using my branch for a few months and it works: https://github.com/giannif/parcel-plugin-static-files-copy

**Comment by elwin013** [comment]
Fixed with #35, closing.

Again - @giannif thank you for contribution :-)
